src/test/groovy/org/cassalog/core/CassalogTest.groovy

```diff
@@ -104,6 +105,28 @@ class CassalogTest extends CassalogBaseTest {
     assertChangeSetEquals(rows[1], new ChangeSet(version: change2Id, author: 'admin', description: 'test'))
   }
 
+  @Test
```

Comment:
There is the ```resetSchema``` method in CassalogBaseTest.groovy that is called in each test. Do you have something else in mind?